tutorials/experiment_design_01.ipynb

```diff
@@ -234,8 +233,8 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "sine_y = util.sine(f=Q_(1, \"Hz\"), amp=Q_([[0, 1, 0]], \"mm\"))\n",
-    "coords = util.lcs_coords_from_ts(sine_y, time)\n",
+    "sine_y = sine(f=Q_(1, \"Hz\"), amp=Q_([[0, 1, 0]], \"mm\"))\n",
+    "coords = lcs_coords_from_ts(sine_y, time)\n",
```

Comment:
```suggestion
    "coords = sine(f=Q_(1, \"Hz\"), amp=Q_([[0, 1, 0]], \"mm\"))\n",
```

Think this should work now, since the LCS now supports `TimeSeries` as coords